src/frontend/src/components/Banner/Banner.css

```diff
@@ -0,0 +1,33 @@
+.heroBanner {
+  background-color: black;
+  height: 100vh;
+}
+
+.bannerImg {
+  background-image: url(../../images/hero-banner.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: scroll;
+  background-size: cover;
+  height: 100vh;
+  opacity: 0.7;
```

Comment:
Slight nit, could you change the opacity anywhere between 0.3-0.5? The image is a little too bright